samples/event_manager_profiler/README.rst

```diff
@@ -16,15 +16,15 @@ Overview
 The sample application consists of two modules that communicate using events:
 
 Module a (``module_a.c``):
-  This module waits for a configuration event (which is sent by ``main.c``).
-  After receiving this event, it starts sending periodic event one sec event to module b.
-  The module also has a counter which is inreased before one sec event is send and which value is transmited in the event.
-  When the module receives a five sec event from the module b, it zeros counter and simulates work for 100ms by sleeping.
+  This module waits for a configuration event sent by ``main.c``.
```

Comment:
Maybe we could name modules using more user-friendly names that provide some information about what given module does ?

Reply:
It is easier to refer to the modules in that case

Reply:
In my opinion if we have only two modules + main naming them with some unique name is more confusing because user will start to wonder which modules are necessary for every  application (like main) and which are only for sample (all other).